### ISSUE TITLE
alt-shift-down/up has duplicate actions - focus next/previous cell and copy line up/down

### ISSUE THREAD

**Issue by awconway** [observablehq/feedback#410]
**Describe the bug**
alt-shift-down/up has duplicate actions - focus next/previous cell and copy line up/down

On windows this shortcut only focuses previous/next  cell

**To Reproduce**
See https://observablehq.com/@observablehq/keyboard-shortcuts

**Expected behavior**
There should be separate shortcuts. Preferably alt-shift-up/down should be for copying a line up/down like other text editors and a separate shortcut used for focus next/previous cell.

**Desktop (please complete the following information):**
 - Windows
 - Edge (Version 101.0.1210.32 (Official build) (64-bit)), chrome Version 100.0.4896.127 (Official Build) (64-bit)

**Comment by mootari** [comment]
Shortcuts work differently depending on wether you're in edit mode (i.e., typing text), or focus mode (where you'll see a blue outline around a cell). You can switch from edit mode to focus mode by pressing Escape, and switch back to edit mode by pressing Enter.

From your description it seems that you cannot duplicate lines while in edit mode?

**Comment by awconway** [comment]
That's correct. alt-shift-down/up (in edit mode) focuses to next/previous cell on my windows computer.
